vita3k/gui/src/archive_install_dialog.cpp

```diff
@@ -65,9 +65,9 @@ void draw_archive_install_dialog(GuiState &gui, EmuEnvState &emuenv) {
     };
     std::lock_guard<std::mutex> lock(install_mutex);
 
-    auto lang = gui.lang.install_dialog.archive_install;
-    auto indicator = gui.lang.indicator;
-    auto common = emuenv.common_dialog.lang.common;
+    auto &lang = gui.lang.install_dialog.archive_install;
+    auto &indicator = gui.lang.indicator;
+    auto &common = emuenv.common_dialog.lang.common;
```

Comment:
```suggestion
    const auto &lang = gui.lang.install_dialog.archive_install;
    const auto &indicator = gui.lang.indicator;
    const auto &common = emuenv.common_dialog.lang.common;
```
Same in other places. Replacing variable to non-const reference could have unpredictable consequences

Reply:
Am I right in thinking that it can no longer use map::operator[] and need to use map::at and rewrite all?
Copy could be better?